### PR TITLE
Major overhaul of chronometer script

### PR DESCRIPTION
**Pull request by PromoFaux**
Added -j option to output stats as JSON formatted string. If run with no option, then chronometer runs as normal.

Fixed calculations to match those on the web dashboard.

Total number of domains now takes whether or not IPv6 is used into consideration.

As this will output a JSON string, this can also be consumed by <code>index.php</code>  AND <code>api.php</code> on the web dashboard.

i.e Change <code>api.php</code> to simply read:
~~<code><?php echo exec("/usr/local/bin/chronometer.sh -j'}"); ?></code>~~
<code><?php echo exec("/usr/local/bin/chronometer.sh -j"); ?></code> (corrected)
Though I don't know anything about PHP, so someone may wish to correct that.

Similarly <code>index.php</code> can read the output of this script and parse the JSON to populate the dashboard.

Plus, think of all the other crazy things you could do with the data. 

image

### CODE REVIEW

File: advanced/Scripts/chronometer.sh [hunk 1]
```diff
@@ -1,35 +1,135 @@
...
-	clear
-	# Displays a colorful Pi-hole logo
-	toilet -f small -F gay Pi-hole
-	echo "        $(ifconfig eth0 | awk '/inet addr/ {print $2}' | cut -d':' -f2)"
-	echo ""
-	uptime | cut -d' ' -f11-
-	echo "-------------------------------"
-	# Uncomment to continually read the log file and display the current domain being blocked
-	#tail -f /var/log/pihole.log | awk '/\/etc\/pihole\/gravity.list/ {if ($7 != "address" && $7 != "name" && $7 != "/etc/pihole/gravity.list") print $7; else;}'
+# (at your opercentBlockedTodayion) any later version.
```
Comment (Mcat12) [inline advanced/Scripts/chronometer.sh]: opercentBlockedTodayion? You coined a new word without even noticing! :smile:

**Comment by Mcat12** [comment]
In your comment about changing api.php, I don't think you need ```}'``` in the string.

File: advanced/Scripts/chronometer.sh (hunk 1 above)
Comment (PromoFaux) [inline advanced/Scripts/chronometer.sh]: fixed

**Comment by PromoFaux** [comment]
Currently api.php a JSON string like this:
<code>{"domains_being_blocked":"136774","dns_queries_today":"6114","ads_blocked_today":"764","ads_percentage_today":12.49591102388}</code>

and this script change outputs this:
<code>{"domains_being_blocked":"68387","dns_queries_today":"6095","ads_blocked_today":"764","ads_percentage_today":"12.53"}</code>

**Comment by Mcat12** [comment]
What's the difference?

**Comment by PromoFaux** [comment]
Oh, wait, I see you're referencing my php psuedocode!

**Comment by Mcat12** [comment]
It should be ```<?php echo exec("/usr/local/bin/chronometer.sh -j"); ?>``` (Without the ```{'```)

**Comment by PromoFaux** [comment]
But other than that, valid PHP script?

**Comment by Mcat12** [comment]
I believe so. There's a feature that I think is standard, where you don't need the ```?>``` if there's only php in the file (and ends with *.php), so it could even be shortened to:
```php
<?php
echo exec("/usr/local/bin/chronometer.sh -j");
```

**Comment by PromoFaux** [comment]
I was brave and tried the PHP changes myself!

**Comment by Mcat12** [comment]
Do they work?

**Comment by PromoFaux** [comment]
Of course they work, I would not commit untested code (anymore!!)

**Pull request merged**